services/backend/src/routes/protected_areas.js

```diff
@@ -26,6 +26,31 @@ module.exports = (errorHandler, paService) => {
       })
   )));
 
+  /**
+   * @apiGroup geofence_pa
+   * @api {get} /pa/categories/binary_protected/:binary_protected listCategoriesByBinaryProtected
+   * @apiName listCategoriesByBinaryProtected
+   * @apiVersion 0.1.0
+   * @apiDescription
+   * List available protected area categories for the given binary protected values
+   *
```

Comment:
Sorry, forgot to mention, missing param documentation

Reply:
and please specify that is a list separated by `&`

Reply:
Done

Reply:
Done